### PR TITLE
Add signup link to top of dashboard introduction page

### LINKED ISSUE #4760: Add a "sign up" CTA at/near the top of the Dashboard Introduction docs page
I'm thinking something along the lines of "Ready to get started? Sign up for the Cypress Dashboard to get X, Y, and Z. <CTA>"

### PR DESCRIPTION
**Pull request by debrisapron**
Closes #4760 

What do we think of this? Questions:

- Do we want it before or after listing the benefits?
- Should we add some custom design element to make it pop more? I thought about putting it in a green "Premium" callout block but it's right underneath another callout (for the RWA) and I think it would be too busy

### CODE REVIEW

**Comment by debrisapron** [comment]
New content here: https://cypress-documentation-9cq5c8ylu-cypress-io.vercel.app/guides/dashboard/introduction#Getting-Started

**Comment by pstakoun** [comment]
maybe could try a green banner at the top under the top "Dashboard" heading?not sure if that would be too prominent. @mattvCypress what do you think?

**Comment by mattvCypress** [comment]
> 

I think that's a fair recommendation. I had to hunt for it on the page.

**Comment by debrisapron** [comment]
New content now at top of page: https://cypress-documentation-6af70zzex-cypress-io.vercel.app/guides/dashboard/introduction

File: content/guides/dashboard/introduction.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 title: Dashboard
 ---
 
+<Alert type="success">
+
+<strong class="alert-header"><Icon name="star"></Icon> Get Started</strong>
+
+Whenever you're ready you can
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:9]:
```suggestion
To get started with the Cypress Dashboard,
```

File: content/guides/dashboard/introduction.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+[sign up here](https://dashboard.cypress.io/signup) for our
+[free plan](https://www.cypress.io/pricing) which gives you up to 3 users and
```
Comment (pstakoun) [inline content/guides/dashboard/introduction.md:11]:
```suggestion
[free plan](https://www.cypress.io/pricing), which gives you up to 3 users and
```

**Review by pstakoun — APPROVED** [review]

**Pull request merged**